docs/tools/flow-cli/flix.md

```diff
@@ -38,21 +77,20 @@ For a list of all templates, check out the [FLIX template repository](https://gi
 
 To generate a FLIX, see the [FLIX CLI readme](https://github.com/onflow/flow-interaction-template-tools/tree/master/cli).
 
-## Arguments
+## Arguments for both execute and package
 - Name: `argument`
-- Valid inputs: valid [cadence values](../../cadence/json-cadence-spec.md)
-  matching argument type in script code.
+- Valid input: valid [FLIX](https://github.com/onflow/flips/blob/main/application/20220503-interaction-templates.md)  
 
-Input arguments values matching corresponding types in the source code and passed in the same order.
-You can pass a `nil` value to optional arguments by executing the flow script like this: `flow scripts execute script.cdc nil`.
+Input argument value matching corresponding types in the source code and passed in the same order.
+You can pass a `nil` value to optional arguments by executing the flow flix execute script like this: `flow flix execute template.json nil`.
 
-## Flags
+## Flags for executing transactions and scripts
```

Comment:
title case

Reply:
not following "title case", Flags for executing transactions and script is the title

Reply:
See this comment: https://github.com/onflow/docs/pull/431#discussion_r1379032567